ts/Extensions/DashboardPlugins/DataGridComponent.ts

```diff
@@ -270,6 +238,55 @@ class DataGridComponent extends Component {
         this.contentElement.appendChild(this.dataGridContainer);
         this.hasLoaded = true;
 
+        if (
+            this.connector &&
+            !this.connectorListeners.length
+        ) {
+            const connectorListeners = this.connectorListeners;
+            // this.on('tableChanged', (): void => this.updateSeries());
+
+            // Reload the store when polling.
+            connectorListeners.push(this.connector
+                .on('afterLoad', (e: DataConnector.Event): void => {
+                    if (e.table && this.connector) {
+                        this.connector.table.setColumns(e.table.getColumns());
+                    }
+                })
+            );
+
+            // Update the DataGrid when store changed.
+            connectorListeners.push(this.connector.table
+                .on('afterSetCell', (e: any): void => {
+                    const dataGrid = this.dataGrid;
+                    let shouldUpdateTheGrid = true;
+
+                    if (dataGrid) {
+                        const row = dataGrid.rowElements[e.rowIndex],
+                            cells = Array.prototype.slice.call(row.childNodes);
+
+                        cells.forEach((cell: HTMLElement): void => {
+                            if (cell.childElementCount > 0) {
+                                const input =
+                                    cell.childNodes[0] as HTMLInputElement,
+                                    convertedInputValue =
+                                        typeof e.cellValue === 'string' ?
+                                            input.value :
+                                            +input.value;
+
+                                if (cell.dataset.columnName === e.columnName &&
+                                    convertedInputValue === e.cellValue
```

Comment:
Is not always true? Because you create value (of convertedInputValue) from e.cellValue

Reply:
I only copied this part of code from constructor, to be fired also on redraw, so I didn't analyse it at all. Might be, but I'd leave it as it is tbh 😅